src/api/app/assets/stylesheets/webui2/groups.scss

```diff
@@ -2,4 +2,9 @@
   .group-user {
     min-width: 26rem;
   }
+
+  a.user-realname {
```

Comment:
Applying a `max-width` to the `.group-user` class above would achieve the same effect. It's also easier to follow what is affecting the width since it would be all under one class.

Reply:
That was the first thing I tried but we'll have the same problem, boxes with different widths, unless we set `min-width`and `max-width` to the same value (equivalent to setting `width`).

But it's true my solution is not nice, let me try something different ...

Reply:
Having boxes with different width is fine. If some are smaller, we can fit more on one row.

What is the goal for having the boxes all with the same width? Because if this is what we want here, setting the max-width on an element inside the `.group-user` container is not obvious. We might as well set the `width` to a specific value. I would go against this, I believe we should take advantage of the flexibiliy provided by the flexbox (we use `.d-flex` on the container of those `.group-user` containers)

Reply:
> What is the goal for having the boxes all with the same width?

Let's say... harmony? For me it looks chaotic with different widths, but perhaps it's a very personal feeling. So, I'll keep it like it was before.

Reply:
Boxes, the same size are way easier to process because I don't have to notice the change in 'rhythm

Reply:
Good point. Let's set the width on the `.group-user` container then.